packages/compass-web/scripts/electron-proxy.js

```diff
@@ -282,6 +360,10 @@ proxyWebServer.use('/authenticate', async (req, res) => {
 
   try {
     const { projectId } = await atlasCloudAuthenticator.authenticate();
+    // Start issuing the cert to save some time when signing in
+    void atlasCloudAuthenticator.getX509Cert().catch(() => {
+      // ignore errors
```

Comment:
Similar comment to elsewhere. Should this at least debug log?

Reply:
Will add one to the get method itself, thanks 👍